tests/translator/input/api_with_resource_policy_global.yaml

```diff
@@ -12,27 +12,22 @@ Globals:
           }]
 
 Resources:
- ExplicitApi:
+  ImplicitApiFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: s3://sam-demo-bucket/member_portal.zip
+      Handler: index.gethtml
+      Runtime: nodejs4.3
+      Events:
+        GetHtml:
+          Type: Api
+          Properties:
+            RestApiId:
+              Ref: ExplicitApi
+            Path: /
+            Method: get
+
+  ExplicitApi:
     Type: AWS::Serverless::Api
     Properties:
       StageName: Prod
-      DefinitionBody: {
```

Comment:
You removed the explicit definition body, but it might be nice to have one test that has an explicit definition body that we can test with.